develop.sh

```diff
@@ -19,8 +21,7 @@ function invoke_manage {
 
 function open_psql_shell {
     invoke_docker_compose run --rm web psql \
-            -U listenbrainz  \
-            -h db listenbrainz
+        $POSTGRES_ADMIN_LB_URI
```

Comment:
can we just do `psql -U listenbrainz -h db` here instead of duplicating the variable from config.py? Is there a way to not make this change at all?

Reply:
If we keep the old setup it will prompt us to add password everytime

Reply:
Actually, there is `-p`, we could set the PGPASSWORD env variable instead...